eth2util/keystore/keystore.go

```diff
@@ -140,24 +141,19 @@ func encrypt(secret *bls_sig.SecretKey, password string, random io.Reader) (keys
 	}
 
 	return keystore{
-		Crypto:  fields,
-		ID:      uuid(random),
-		Version: encryptor.Version(),
-		Pubkey:  hex.EncodeToString(pubKeyBytes),
-		Name:    encryptor.Name(),
+		Crypto:      fields,
+		Description: "",
```

Comment:
add comment why empty maybe? (note you do not need to put empty since that is the default)